README.md

```diff
@@ -87,7 +87,7 @@ Where available commands are:
 | files        | display installed files of package(s) |
 | help         | get information about command usage |
 | install      | install package(s) along with dependencies after prompting for confirmation |
-| list         | available, installed, compatible or incompatible packages |
+| list         | available, compatible, incompatible, essential, installed packages |
```

Comment:
List alphabetically?

Reply:
I put those together intentionally so that `compatible` and `incompatible` were logically next to each other.

Reply:
How do you feel about that?

Reply:
Well, you could put available and installed together logically too.  I guess it doesn't matter.